code/game/machinery/alarm.dm

```diff
@@ -1214,6 +1214,8 @@ FIRE ALARM
 	return ..()
 
 /obj/machinery/firealarm/CtrlClick(var/mob/user)
+	if (ismouse(user)) // Squeak
```

Comment:
a dex check might make more sense, or ishigherbeing

Reply:
Wouldn't that make monkeys unable to use it ? Given that `attack_paw` calls `attack_hand`, it seems they were intended to use it.

Reply:
Higherbeing allows monkeys with methylin

You could also do higher being or monkey. To eliminate use by cows.

Reply:
Alright